File: modele_classes_csharp/StatusHistory.cs
using System;

namespace Moov.Facturation.Utilisateurs
{
    /// <summary>
    /// Historique des changements de statut des utilisateurs
    /// </summary>
    public class StatusHistory
    {
        // Attributs
        public string Id { get; set; }
        public string OldStatus { get; set; }
        public string NewStatus { get; set; }
        public DateTime ChangedAt { get; set; }
        
        // Relations
        public User User { get; set; }
    }
}

File: modele_classes_java/StatusHistory.java
package com.moov.facturation.utilisateurs;

import java.util.Date;

/**
 * Historique des changements de statut utilisateur
 */
public class StatusHistory {
    
    // Attributs
    private String id;
    private String oldStatus;
    private String newStatus;
    private Date changedAt;
    
    // Relations
    private User user;
    private User changedBy;
    
    // Getters et Setters
    public String getId() { return id; }
    public void setId(String id) { this.id = id; }
    
    public String getOldStatus() { return oldStatus; }
    public void setOldStatus(String oldStatus) { this.oldStatus = oldStatus; }
    
    public String getNewStatus() { return newStatus; }
    public void setNewStatus(String newStatus) { this.newStatus = newStatus; }
    
    public Date getChangedAt() { return changedAt; }
    public void setChangedAt(Date changedAt) { this.changedAt = changedAt; }
    
    public User getUser() { return user; }
    public void setUser(User user) { this.user = user; }
    
    public User getChangedBy() { return changedBy; }
    public void setChangedBy(User changedBy) { this.changedBy = changedBy; }
}
